lighthouse-core/audits/dobetterweb/script-blocking-first-paint.js

```diff
@@ -33,7 +33,7 @@ class ScriptBlockingFirstPaint extends Audit {
     return {
       category: 'Performance',
       name: 'script-blocking-first-paint',
-      description: 'Site does not use <script> in head that delays first paint',
+      description: 'Avoids `<script>` in head that delay first paint',
```

Comment:
I read "Avoids `<script> in head...` as "Avoids scripts in head....", but I'll change it back to "Avoids `<script>` in head that delays first paint"